### PR TITLE
Support for env tag

### PR DESCRIPTION
**Pull request by pickgr**

### CODE REVIEW

**Comment by pickgr** [comment]
@blakeneyops - please review

**Comment by cristobal23** [comment]
You can use branch, commit, or tag as ref.  I just cut release [Version 0.2.1](https://github.com/WhistleLabs/terraform-aws-asg/releases/tag/v0.2.1)

**Comment by pickgr** [comment]
@cristobal23 - as we discussed, feel free to include this in your next release if you get to it before I do.  I just haven't really tested this yet.  Mainly want to make sure there aren't any negative effects.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The group module will provision security group, launch configuration and auto sc
 **Resource labels**
 * `stack_item_label` - Short form identifier for this stack. This value is used to create the **Name** resource tag for resources created by this stack item, and also serves as a unique key for re-use.
 * `stack_item_fullname` - Long form descriptive name for this stack item. This value is used to create the **application** resource tag for resources created by this stack item.
+* `env` - Descriptive name for this stack's environment. This value is used to create the **tag** resource tag for resources created by this stack item.
```
Comment (cristobal23) [inline README.md]: Is this variable required or optional?

File: group/asg/variables.tf [hunk 2]
```diff
@@ -9,6 +9,10 @@ variable "stack_item_fullname" {
   type = "string"
 }
 
+variable "env" {
```
Comment (cristobal23) [inline group/asg/variables.tf]: Does this variable have a default value?

Comment (cristobal23) [inline group/asg/variables.tf]: yes. it is empty string ''

**Pull request merged**